pkg/config/sds/config.go

```diff
@@ -8,14 +8,17 @@ import (
 
 func DefaultSdsServerConfig() *SdsServerConfig {
 	return &SdsServerConfig{
-		GrpcPort: 5677,
+		GrpcPort:       5677,
+		PublicHostPort: "",
 	}
 }
 
 // Envoy SDS server configuration
 type SdsServerConfig struct {
 	// Port of GRPC server that Envoy connects to
 	GrpcPort int `yaml:"grpcPort" envconfig:"kuma_sds_server_grpc_port"`
+	// Public HostPort to reach SDS server
+	PublicHostPort string `yaml:"publicHostPort" envconfig:"kuma_sds_server_public_host_port"`
```

Comment:
Can we include this in the catalog instead? Just like MADS server.
I think there should be also XDS server there, because my guess is that you would also like to override this.

Reply:
yes, i can move it to catalog seems to be a better place to manage all overwrites. 
XDS is derived via AdvertisedHostname. let me know if you are ok to derive from this new variable.

Reply:
Just realised, XDS is a part of bootstrapServer config. Probably we can keep it that way. Let me know.